### PR TITLE
Lambda expressions: More attribute examples and style edits

### PR DESCRIPTION
**Pull request by pkulikov**
Added example of attributes applied to the input parameters and a return value.
Also, the last paragraph of the **Attributes** section looks to be important enough for a highlight.
Furthermore, removed future tense and reduced amount of passive voice.

### CODE REVIEW

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: when @BillWagner sees this and is ready. Thank you

**Comment by IEvangelist** [comment]
Hey, @pkulikov - I think you may have broken a bookmark as part of this PR:

**Line 82: [Warning]** Cannot find bookmark `#natural-type-for-lambda-expressions` in:
_docs/csharp/language-reference/operators/lambda-expressions.md_ 
> did you mean `#natural-type-of-a-lambda-expression`?

**Line 83: [Warning]** Cannot find bookmark `#declared-return-type` in:
_docs/csharp/language-reference/operators/lambda-expressions.md_
> did you mean `#explicit-return-type`?

File: docs/csharp/language-reference/operators/lambda-expressions.md [hunk 1]
```diff
@@ -250,32 +250,40 @@ The compiler can't infer a parameter type for `s`. When the compiler can't infer
...
+
+```csharp
+var sum = ([Example(1)] int a, [Example(2), Example(3)] int b) => a + b;
+var inc = [return: Example(1)] (int s) => s++;
+```
+
+As the preceding examples show, you must parenthesize the input parameters when you add attributes to a lambda expression or its parameters.
 
-Lambda expressions are invoked through the underlying delegate type. That is different than methods and local functions. The delegate's `Invoke` method won't check attributes on the lambda expression. Attributes don't have any effect when the lambda expression is invoked. Attributes on lambda expressions are useful for code analysis, and can be discovered via reflection. One consequence of this decision is the <xref:System.Diagnostics.ConditionalAttribute?displayProperty=nameWithType> cannot be applied to a lambda expression.
+> [!NOTE]
```
Comment (BillWagner) [inline docs/csharp/language-reference/operators/lambda-expressions.md:285]: I agree this should be called out. Maybe change from Note to Important?

```suggestion
> [!IMPORTANT]
```

**Review by BillWagner — COMMENTED** [review]
I'll agree this is ready to go, once the warnings are resolved. (I also think it's due to the changed anchor tags).

I had one optional suggestion.

Thanks again @pkulikov

**Comment by pkulikov** [comment]
@IEvangelist thank you!

**Pull request merged**